packages/react/src/QuestionnaireForm/QuestionnaireFormItem/QuestionnaireFormItem.tsx

```diff
@@ -413,6 +413,9 @@ function typedValueToString(typedValue: TypedValue | undefined): string | undefi
   if (typedValue.type === 'Coding') {
     return formatCoding(typedValue.value);
   }
+  if (typedValue.type === 'Reference') {
```

Comment:
We need to add a check here: previously in a DropDownChoice it would show 
<img width="710" alt="Screenshot 2023-10-27 at 2 56 18 PM" src="https://github.com/medplum/medplum/assets/6913745/4bca0bac-986b-4091-86f1-24391f843327">

Not only that, the initialValue wouldn't render correctly